pkg/operatorsource/succeeded.go

```diff
@@ -68,6 +69,9 @@ func (r *succeededReconciler) Reconcile(ctx context.Context, in *v1.OperatorSour
 
 	err = defaults.New(defaults.GetGlobalDefinitions(), operatorhub.GetSingleton().Get()).Ensure(r.client, in.Name)
 
+	// Report the OperatorSource success.
+	status.ReportOperatorSourceSuccess(out.Name)
```

Comment:
This reports that the OperatorSource is syncing properly, but not whether or not the catalogsource can successfully pull from an appregistry (i.e. we can be serving 0 packages and this won't report degraded).

Maybe this is fine? The BZ for this says:

> 1. Make marketplace unable to successfully reconcile one of the default OperatorSources.

can you tell me how to actually do that on a cluster? everything I think of involved disrupting the connection to quay or damaging the catalogsource pod in some way, which doesn't actually affect operatorsource reconciliation.